### PR TITLE
Remove default settings

### PR DESCRIPTION
**Pull request by gavinleeedwards**
Remove any default settings from settings.py and check on startup for any missing values.
Change README to remove defaults and give examples of all environment variables.

### CODE REVIEW

File: app/consumer.py [hunk 1]
```diff
@@ -23,6 +24,22 @@ def get_delivery_count_from_properties(properties):
...
+def check_default_env_vars():
+
+    env_vars = ["LOGGING_LEVEL", "RECEIPT_HOST", "RECEIPT_PATH", "RECEIPT_USER",
+                "RECEIPT_PASS", "RABBITMQ_QUARANTINE_QUEUE", "RECEIPT_RRM_QUEUE", "RABBITMQ_EXCHANGE",
+                "SDX_RECEIPT_RRM_SECRET", "RABBITMQ_HOST", "RABBITMQ_HOST2", "RABBITMQ_PORT",
+                "RABBITMQ_PORT2", "RABBITMQ_DEFAULT_USER", "RABBITMQ_DEFAULT_PASS", "RABBITMQ_DEFAULT_VHOST"]
+
+    for i in env_vars:
+        if os.getenv(i) is None:
+            logger.error("No ", i, "env var supplied")
```
Comment (necrophonic) [inline app/consumer.py]: Might be cleaner to pass key/value pairs, like
```
logger.error("Required env var missing", missing=i)
```
Or something similar

File: app/consumer.py [hunk 2]
```diff
@@ -23,6 +24,32 @@ def get_delivery_count_from_properties(properties):
...
+def check_default_env_vars():
+
+    env_vars = ["RECEIPT_HOST", "RECEIPT_PATH", "RECEIPT_USER",
+                "RECEIPT_PASS", "RABBITMQ_QUARANTINE_QUEUE", "RECEIPT_RRM_QUEUE", "RABBITMQ_EXCHANGE",
+                "SDX_RECEIPT_RRM_SECRET", "RABBITMQ_HOST", "RABBITMQ_HOST2", "RABBITMQ_PORT",
+                "RABBITMQ_PORT2", "RABBITMQ_DEFAULT_USER", "RABBITMQ_DEFAULT_PASS", "RABBITMQ_DEFAULT_VHOST"]
+
+    missing_env_var = False
+
+    for i in env_vars:
```
Comment (necrophonic) [inline app/consumer.py:42]: Might be cleaner to do the check in the block and list the env var as a separate value to the log:
```python
for var in env_vars:
    if not os.getenv(var):
        logger.error("Failed to start", error="Missing env var", var=var)
        missing_env_var = True
```

**Comment by JamesGardiner** [comment]
@tundish - have you looked at this service as part of your current story card?

**Comment by necrophonic** [comment]
Closing for now

**Pull request closed without merging**